CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+# [0.19.0](https://github.com/dashevo/js-dpp/compare/v0.18.0...v0.19.0) (2021-04-30)
+
+
+### Features
+
+* add data triggers for feature flags documents ([#297](https://github.com/dashevo/js-dpp/issues/297))
+* ChainLock Asset Lock proof ([#296](https://github.com/dashevo/js-dpp/issues/296))
+* use `verifyInstantLock` instead of `fetchSMLStore` ([#294](https://github.com/dashevo/js-dpp/issues/294))
+
+
+### Bug Fixes
+
+* height check was missing in feature flag data trigger ([#302](https://github.com/dashevo/js-dpp/issues/302))
+
+
```

Comment:
Please describe breaking changes.

Reply:
"New `ChainAssetLock` proof type was added" is not breaking change.

* `AssetLock` class was removed.
* `InstantAssetLockProof` requires `transaction` and `outputIndex` property.
* `IdentityCreateTransition` schema changed. `assetLock` property renamed to `assetLockProof` and expect `InstantAssetLockProof` or `ChainAssetLockProof`. `transaction` and `outputIndex` properties are removed.
* `IdentityTopUpTransition` schema changed. `assetLock` property renamed to `assetLockProof` and expect `InstantAssetLockProof` or `ChainAssetLockProof`. `transaction` and `outputIndex` properties are removed.